src/main/java/seedu/stockpal/data/product/Product.java

```diff
@@ -59,15 +60,21 @@ public boolean isPidMatch(Pid pid) {
     }
 
     public void increaseQuantity(Integer amountToChange) {
-        quantity.updateIncreaseQuantity(amountToChange);
+        try {
+            quantity.updateIncreaseQuantity(amountToChange);
+            Ui.printToScreen("Quantity updated. " + quantity.toString());
+        } catch (InventoryQuantityOverflowException IQOE) {
+            Ui.printToScreen("Overflow detected. No change to quantity. " + quantity.toString());
+        }
+
     }
 
     public void decreaseQuantity(Integer amountToChange) {
         try {
             quantity.updateDecreaseQuantity(amountToChange);
-            Ui.printToScreen("Quantity updated. " + quantity);
-        } catch (InsufficientAmountException e) {
-            Ui.printToScreen("Insufficient amount in inventory. No change to quantity." + quantity);
+            Ui.printToScreen("Quantity updated. " + quantity.toString());
+        } catch (InsufficientAmountException IAE) {

```

Comment:
Uppercase variables are usually constants, reconsider if this is applicable here?

Reply:
Noted and edited.